src/storage/patch/SparqlUpdatePatchHandler.ts

```diff
@@ -1,8 +1,10 @@
 import type { Readable } from 'stream';
+import type { ActorInitSparql } from '@comunica/actor-init-sparql';
+import { newEngine } from '@comunica/actor-init-sparql';
+import type { IQueryResultUpdate } from '@comunica/actor-init-sparql/lib/ActorInitSparql-browser';
```

Comment:
Oh, we probably want this one to be available in this export as well: https://github.com/comunica/comunica/blob/master/packages/actor-init-sparql/index.ts